_posts/2019-04-09-sr2019-tlc-crowned-champion.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: news
+title: The Ladies' College Wins Student Robotics 2019
```

Comment:
I think their full name is "The Ladies' College, Guernsey"
```suggestion
title: The Ladies' College, Guernsey Wins Student Robotics 2019
```

(also elsewhere, not explicitly noted)

Reply:
https://github.com/srobo/website/pull/148/commits/ee104ec184843d20a545e434cf0093575b141a17